README.md

```diff
@@ -1 +1,27 @@
-# Wall Of Memes
+CU-News
+===================
+
+CU-News is a web app that displays news articles from multiple reputable sources and finds which news articles Carleton students care about, with analysis of trends.
+
+![screenshot of CU-News](http://i.imgur.com/L2xU1LM.png)
+
+
+----------
+
+Voting System
+-------------
+
+Each article has key phrases extracted from it and upvoting a article adds points to all key phrases associated to that article. Popular key phrases show us issues for topics people care most about.
+
+----------
+
+Analytics
+-------------------
+
+The popularity of all key phrases can be graphed over time and can be compared to each other
```

Comment:
Revised sentence: 

```The popularity of all key phrases can be graphed over time, and key phrases can be compared to each other.```

You might want to add 'graphically' or similar to the end of this sentence, depending on how we will compare them.